config/__tests__/__snapshots__/config.ci.test.js.snap

```diff
@@ -32,14 +32,15 @@ Array [
   "client.majorSubjectAreas.epidemiology-global-health",
   "client.majorSubjectAreas.evolutionary-biology",
   "client.majorSubjectAreas.genetics-genomics",
-  "client.majorSubjectAreas.human-biology-medicine",
```

Comment:
Are we certain we want to remove `human-biology-medicine` and accept the impact, however small, on any manuscripts in various phases of submission?